lib/config.js

```diff
@@ -48,7 +48,12 @@ config.mongodb.connectOptions = {
     journal: true
   },
   server: {
-    auto_reconnect: true
+    socketOptions: {
+      autoReconnect: true, // reconnect on connection failure
+      keepAlive: 15000, // TCP keep-alive every 15 seconds (useful for VPN'd connections)
```

Comment:
I'm not sure the `keepAlive` setting does what the comment says. The documentation says "The number of milliseconds to wait before initiating keepAlive on the TCP socket." Which means maybe we don't want this. The default is 0.